### DISCUSSION TITLE
Missing release notes for v2.1.0

### DISCUSSION THREAD

**Discussion by stevehipwell** [fluent/fluent-bit#7989] [General]
The [release notes](https://fluentbit.io/announcements/v2.1.10/) for [v2.1.0](https://github.com/fluent/fluent-bit/releases/tag/v2.1.10) are missing.

CC @edsiper @patrick-stephens

**Comment by patrick-stephens** [comment]
I'm slopey shouldering this one to @edsiper but I know he's travelling @stevehipwell so may be a little delay @stevehipwell

**Reply by patrick-stephens** [reply]
These are up now @stevehipwell
